lib/ports.dart

```diff
@@ -321,14 +374,15 @@ class SingleResponseChannel<R> {
   SendPort get port => _receivePort.sendPort;
 
   /// Future completed by the first value sent to [port].
-  Future<R> get result => _completer.future;
+  /// Null is only possible with null timeoutValue and
```

Comment:
Extra empty in the comment line before the second paragraph. Code quotes around `[timeoutValue]`.

Reply:
Removed since it seems to be an incomplete comment.